Tools/clinic/clinic.py

```diff
@@ -4594,10 +4594,6 @@ def parse_arg(self, argname: str, displayname: str, *, limited_capi: bool) -> st
                 if (PyObject_GetBuffer({argname}, &{paramname}, PyBUF_SIMPLE) != 0) {{{{
                     goto exit;
                 }}}}
-                if (!PyBuffer_IsContiguous(&{paramname}, 'C')) {{{{
-                    {bad_argument}
-                    goto exit;
-                }}}}
```

Comment:
FTR, this (and the whole function) was added in gh-9689 (@serhiy-storchaka, 2018).

Reply:
It may be helpful to add a comment indicating that the buffer is guaranteed to be contiguous (due to `PyBUF_SIMPLE`) for the format units for buffers.  A single comment at the top of the clinic function (rather than in the generated code) would probably be cleanest.